### PR TITLE
update go-framed-msgpack-rpc and start using ConnectionOpts

### PR DESCRIPTION
**Pull request by oconnor663**

### CODE REVIEW

**Comment by oconnor663** [comment]
CircleCI's build failure is the same issue as in https://github.com/keybase/client/pull/5356#issuecomment-271935552

**Comment by mmaxim** [comment]
I imagine they might want to keep the exponential backoff (with a shorter max interval), but I'll let @strib make that call,

**Comment by akalin-keybase** [comment]
This lgtm, leaving whether or not to use exponential backoff to @strib.

File: libkbfs/constants.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import "time"
+
+// RPCReconnectInterval specifies the between reconnect attempts for RPC Connections.
```
Comment (strib) [inline libkbfs/constants.go]: Missing word somewhere in "specifies the between".

Comment (oconnor663) [inline libkbfs/constants.go]: https://github.com/keybase/kbfs/pull/654

**Pull request merged**